python_modules/libraries/dagster-aws/dagster_aws/pipes/message_readers.py

```diff
@@ -379,7 +405,7 @@ def download_messages(
         if cursor is not None:
             params["nextToken"] = cursor
 
-        response = self.client.get_log_events(**params)
+        response = get_log_events(client=self.client, **params)
```

Comment:
I suggest we wrap `self.client.get_log_events` in `dagster._utils.backoff` instead - it's already written :)

I also don't think we need exponential backoff here (since we aren't doing anything massively parallel).

Reply:
ah wasn't aware of that function, nice - updated